frontend/device/remarkable/device.lua

```diff
@@ -5,21 +5,47 @@ local function yes() return true end
 local function no() return false end
 
 local Remarkable = Generic:new{
-    model = "reMarkable",
     isRemarkable = yes,
+    model = "reMarkable",
     hasKeys = yes,
     needsScreenRefreshAfterResume = no,
     hasOTAUpdates = yes,
     canReboot = yes,
     canPowerOff = yes,
     isTouchDevice = yes,
+    invertX = yes,
```

Comment:
This line should be added to generic?

Reply:
We've never done so for platform-specific tweaks like that (e.g., `touch_mirrored_x` on Kobo/Cervantes).

Reply:
Not quite that generic, but it looked like an incorrect leftover at first glance based on your earlier comments.

Reply:
i'll push  it down and rename it